src/Helper/WorkQueue.php

```diff
@@ -1414,6 +1433,7 @@ class WorkQueue
         'retentionType',
         'isEhrDataAvailable',
         'latestEhrReceiptTime',
+        'healthDataStream',
```

Comment:
We should always add new columns to the last in CSV export. Could you move this field to the last position?